atlp-rwanda/Avengers-Phantom-FE
# bg(UI):fixing ui designs on dashboard

## What does this PR do?

Dashboard UI components Improvement

## Description of Task to be completed?

- [x] Improved Routes list design
- [x] Improved add route design
- [x] Improved SideBar Items arrangement
- [x] Improved Driver list design
- [x] Improved add driver design
- [x] Improved operator list design
- [x] Improved add operator design
- [x] Improved Bus list design
- [x] Improved add bus design
- [x] Improved update Profile design

WIP

## How should this be manually tested?

step1: clone git repo if not yet done
step2: cd Avengers-Phantom-FE
step3: npm i or npm install to install dependencies
step4: `npm run dev` to start the server
step5: switch to` bg-dashboard-ui-improvement` branch
step6: Login to go to the dashboard with different user accounts to check Routes protection.
step7: Switch to different components by clicking sidebar nav items

<img width="1440" alt="Screen Shot 2022-07-10 at 19 25 54" src="https://user-images.githubusercontent.com/59208992/178155473-5013a20b-9d8d-4457-bb2b-d9432aec7b2d.png">

<img width="1440" alt="Screen Shot 2022-07-10 at 19 28 45" src="https://user-images.githubusercontent.com/59208992/178155529-720fd60e-40dc-4828-98b7-67ed87fdd6e9.png">

<img width="1440" alt="Screen Shot 2022-07-10 at 19 30 11" src="https://user-images.githubusercontent.com/59208992/178155586-634fc345-c7cb-4e74-9294-1352b9220900.png">
